bubble_sort.rb

```diff
@@ -0,0 +1,50 @@
+
+def bubble_sort
+	puts 'Enter number of items'
+
+	total_items = gets.chomp
+
+	items = total_items.to_i
+
+	array = []
+
+	0.upto(items - 1) do |i|
+		array[i] = gets.chomp
+	end
+
+	puts "Total Number of items in the array is:  #{total_items}"
+
+	puts "Youre array is #{array}"
+
+	array.each do
+		i = 0
+		count = 0
+		while i < items - 1
+			array[i], array[i + 1] = array[i + 1], array[i] if array[i] > array[i + 1]
+			i += 1
+			count += 1
+		end
+		break if count.zero?
```

Comment:
- [ ] Kindly remove all the puts/print/chomp statements in this method because they are not needed. 
What is required of you in this method are the following;
  - that the method should accept an array as a parameter.
  - then you perform the sorting logic on that array and only return the value of that array.

Reply:
**Thank You for the suggestion!**
- [x]    Its done now!